### PR TITLE
Add an option like a digging

### PR DESCRIPTION
**Pull request by CyberPatrick**

### CODE REVIEW

File: lib/plugins/place_block.js [hunk 1]
```diff
@@ -22,7 +22,9 @@ function inject (bot, { version }) {
       dy = options.delta.y
       dz = options.delta.z
     }
-    await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), false)
+    if (options.forceLook !== 'ignore') {
+      await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), options.forceLook)
```
Comment (rom1504) [inline lib/plugins/place_block.js:26]: You are passing here something of type string to something that expects a boolean

Comment (CyberPatrick) [reply lib/plugins/place_block.js:26]: Options.forceLook can be: 'ignore', false, true. If it is not 'ignore' - it is boolean. https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/digging.js#L30-L40 - here the same thing.

Comment (rom1504) [reply lib/plugins/place_block.js:26]: Is it handled by the function you're passing it too ?

Comment (CyberPatrick) [reply lib/plugins/place_block.js:26]: No

Comment (Karang) [reply lib/plugins/place_block.js:26]: lookAt isnt called if forceLook isnt a boolean

Comment (CyberPatrick) [reply lib/plugins/place_block.js:26]: So i should add check? 
```
If (options.forceLook !== 'ignore' && typeof options.forceLook !== Boolean) {
    throw new Error('forceLook invalid input')
}
```
Or just in doc add info about types.

**Review by rom1504 — APPROVED** [review]

**Pull request merged**